src/components/paragraph/stanford-media-caption.tsx

```diff
@@ -42,11 +42,15 @@ const StanfordMediaCaption = ({caption, image, videoUrl, link, ...props}: Props)
         </div>
       }
 
-      {link &&
-        <Link href={link.url} className="block text-right" {...link.options?.attributes}>
-          {link.title}
-        </Link>
-      }
+      <div className={'text-right'}>
```

Comment:
It's a bit cleaner to only use the brackets when necessary.
```suggestion
      <div className="text-right">
```

Reply:
Also, this div probably should go inside the `{link &&` statement so it doesn't render if there is no link.